app/auth/routes.py

```diff
@@ -77,6 +77,7 @@ class UserView(Resource):
     @jwt_required
     def get(self):
         current_user = User.query.filter_by(username=get_jwt_identity()).first()
+        from marshmallow import pprint
```

Comment:
Was this leftover from debugging?

Reply:
Yep. You can remove it or I'll update it on my next commit.